includes/class-sensei-posttypes.php

```diff
@@ -387,7 +387,7 @@ public function setup_sensei_message_post_type () {
 			    'map_meta_cap' => true,
 			    'capability_type' => 'question',
 			    'has_archive' => true,
-			    'hierarchical' => false,
```

Comment:
Are messages really hierarchical – do messages have parent messages? What are the implications from this change?